### ISSUE TITLE
Some cue sheets play whole album for every track

### ISSUE THREAD

**Issue by KazimirPodolski** [strawberrymusicplayer/strawberry#552]
For technical issues, questions and feature requests please use the forum on https://forum.strawberrymusicplayer.org/

**Describe the bug**

With some `.cue`s in my collection tracks do not continue to next track, but instead play until the end of album. Then next track plays itself plus the rest of the album, and so on.

**To Reproduce**
See this .cue for example:

```
REM GENRE "Black Metal"
REM DATE 2012
REM DISCID 2C0D1804
REM COMMENT "ExactAudioCopy v1.0b2"
PERFORMER "Elysian Blaze"
TITLE "Blood Geometry"
FILE "Elysian Blaze - Blood Geometry - CD1.flac" WAVE
  TRACK 01 AUDIO
    TITLE "A Choir for Venus "
    PERFORMER "Elysian Blaze"
    ISRC 000000000000
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "The Temple Is Falling "
    PERFORMER "Elysian Blaze"
    ISRC 000000000000
    INDEX 01 04:30:51
  TRACK 03 AUDIO
    TITLE "Sigils That Beckon Death "
    PERFORMER "Elysian Blaze"
    ISRC 000000000000
    INDEX 01 22:55:52
  TRACK 04 AUDIO
    TITLE "Blood Geometry"
    PERFORMER "Elysian Blaze"
    ISRC 000000000000
    INDEX 01 46:50:53
```

**Expected behavior**
Tracks play as they are actually seen.

**Screenshots:**
![image](https://user-images.githubusercontent.com/6647438/95079955-8d4f8680-0741-11eb-9c93-44b24fd5e556.png)
Look at playback time

**System Information:**
 - Operating system: KDE Neon (Ubuntu 20.04.1)
 - Strawberry Version: 0.7.2

**Comment by jonaski** [comment]
I see nothing wrong with that cue file. What is shown for length column on the songs in the playlist? Did you try a rescan?

**Comment by KazimirPodolski** [comment]
Length column is correct - actual tracks length. Rescan did not help.

**Comment by jonaski** [comment]
Can you send me a problematic cue album on e-mail? (e-mail address is listed on my profile).

**Comment by KazimirPodolski** [comment]
Sure.

**Comment by jonaski** [comment]
I can't reproduce any problems, I only played through the first song, the other songs I tested seeking to the end and checked that it proceeds to the next song successfully.

**Comment by KazimirPodolski** [comment]
Strange. I'll try to localize my problem.

**Comment by jonaski** [comment]
Have you set any special settings in the backend settings? I assume it is set to gstreamer? Do you know what version of gstreamer you have?

**Comment by KazimirPodolski** [comment]
Only Replay Gain is on (setting it off doesn't make a difference. Trying to set/unset all settings related to playback). It's gstreamer 1.16.2.

**Comment by jonaski** [comment]
I actually can reproduce this with the replay gain setting on. New replay gain setting won't take affect until you stop/start playback.

**Comment by jonaski** [comment]
This should fix it b6ff7e6, try the devlopment build when available on https://builds.strawberrymusicplayer.org/ 
Should be 0.8.1-2

**Comment by KazimirPodolski** [comment]
Ah, I should have restarted the player after each settings change of course. Thanks for the fix!
